calcs/evol.py

```diff
@@ -32,8 +32,8 @@ def de_dt(e, times, beta, c_0):
     dedt : `array`
         eccentricity evolution
     """
-    dedt = -19/12 * beta/c_0**4 * (e**(29/19)*(1 - e**2)**(3/2)) \
-        / (1+(121/304)*e**2)**(1181/2299)
+    dedt = -19 / 12 * beta / c_0**4 * (e**(-29 / 19) * (1 - e**2)**(3/2)) \
```

Comment:
This was the dedt bug -- `e**(29/19)` instead of `e**(-29/19)`, sneaky little minus sign!

Reply:
good catch!